packages/react-table/src/components/TableComposable/examples/ComposableTableActionsOverflow.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-console */
+import React from 'react';
+import {
+  Button,
+  OverflowMenu,
+  OverflowMenuControl,
+  OverflowMenuContent,
+  OverflowMenuGroup,
+  OverflowMenuItem,
+  OverflowMenuDropdownItem,
+  Dropdown,
+  KebabToggle
+} from '@patternfly/react-core';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+
+interface Repository {
+  name: string;
+  branches: string;
+  prs: string;
+  workspaces: string;
+  lastCommit: string;
+  isMenuOpen: boolean;
+}
+
+export const ComposableTableActions: React.FunctionComponent = () => {
+  // In real usage, this data would come from some external source like an API via props.
+  const repositories: Repository[] = [
+    { name: 'one', branches: 'two', prs: 'a', workspaces: 'four', lastCommit: 'five', isMenuOpen: false },
+    { name: 'a', branches: 'two', prs: 'k', workspaces: 'four', lastCommit: 'five', isMenuOpen: false },
+    { name: 'p', branches: 'two', prs: 'b', workspaces: 'four', lastCommit: 'five', isMenuOpen: false },
+    { name: '4', branches: '2', prs: 'b', workspaces: 'four', lastCommit: 'five', isMenuOpen: false },
+    { name: '5', branches: '2', prs: 'b', workspaces: 'four', lastCommit: 'five', isMenuOpen: false }
+  ];
+
+  const [repos, setRepos] = React.useState(repositories);
+
+  const columnNames = {
+    name: 'Repositories',
+    branches: 'Branches',
+    prs: 'Pull requests',
+    workspaces: 'Workspaces',
+    lastCommit: 'Last commit',
+    overflowAction: 'Overflow action'
+  };
+
+  const dropdownItems = [
+    <OverflowMenuDropdownItem key="item1" isShared>
+      Pimary
+    </OverflowMenuDropdownItem>,
+    <OverflowMenuDropdownItem key="item2" isShared>
+      Secondary
+    </OverflowMenuDropdownItem>,
+    <OverflowMenuDropdownItem key="item3" isShared>
+      Tertiary
+    </OverflowMenuDropdownItem>
+  ];
+
+  return (
+    <React.Fragment>
+      <TableComposable aria-label="Actions table">
+        <Thead>
+          <Tr>
+            <Th>{columnNames.name}</Th>
+            <Th>{columnNames.branches}</Th>
+            <Th>{columnNames.prs}</Th>
+            <Th>{columnNames.workspaces}</Th>
+            <Th>{columnNames.lastCommit}</Th>
+            <Td>{columnNames.overflowAction}</Td>
```

Comment:
If the action table header cell has content, it is a `<th class="pf-c-table__action">`, and in this case I would add `pf-m-fit-content` here. 

Resulting in: `<th class="pf-c-table__action pf-m-fit-content">`

As an aside, `.pf-c-table__action` should be `text-align: right`. https://github.com/patternfly/patternfly/issues/4716

Reply:
https://github.com/patternfly/patternfly/pull/4717

Reply:
Disregard